src/vcs/git/stats.rs

```diff
@@ -0,0 +1,36 @@
+// This file is part of radicle-surf
```

Comment:
How did this end up in the PR? Was it moved?

Reply:
Can you be more specific in what "it" is you're referring to?

Reply:
The `Stats` seem unrelated to the change-seet.

Reply:
I moved them out of `object.rs` unchanged.

As I said in the description:
> The other refactor involves moving the definitions from object.rs into their own sub-modules. It was scaring me that object was a catch all for git things. I prefer having a hierarchy.

Reply:
Sorry didn't get that far, was too excited about the first three paragraphs. :service_dog: